Jellyfin.Api/Controllers/SessionController.cs

```diff
@@ -391,7 +392,7 @@ public class SessionController : BaseJellyfinApiController
             _sessionManager.ReportCapabilities(id, new ClientCapabilities
             {
                 PlayableMediaTypes = RequestHelpers.Split(playableMediaTypes, ',', true),
-                SupportedCommands = RequestHelpers.Split(supportedCommands, ',', true),
+                SupportedCommands = supportedCommands == null ? Array.Empty<GeneralCommandType>() : supportedCommands,
```

Comment:
The binder always returns an array
```suggestion
                SupportedCommands = supportedCommands,
```

Reply:
Maybe null is returned when the parameter isn't set? Have not tested yet

Reply:
The binder returns an empty array when the parameter isn't set, I just tested it :)